go.mod

```diff
@@ -30,6 +30,7 @@ require (
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
 	golang.org/x/sys v0.0.0-20200622214017-ed371f2e16b4
 	gopkg.in/yaml.v2 v2.3.0
+	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776
```

Comment:
Not for this PR, but do you think it's worth us migrating our other YAML code to v3 so we don't have to pull in both versions?

Reply:
Yeah, I think so. I just migrated all of our campaigns code to v3 (two other places in `_test.go` files) and the only other place that uses yaml is in `cmd/src/validate.go`. Ticket: https://github.com/sourcegraph/src-cli/issues/429